### PR TITLE
added text to aboutus

### PR DESCRIPTION
**Pull request by hexflores87**

### CODE REVIEW

File: app/javascript/components/pages/AboutUs.js [hunk 1]
```diff
@@ -62,10 +67,17 @@ const AboutUs = (props) => {
         <CardBody>
           <CardTitle tag="h5">Jonathan Sanchez</CardTitle>
           <CardSubtitle tag="h6" className="mb-2 text-muted">
-          Full Stack Developer
+            Full Stack Developer
           </CardSubtitle>
           <CardText>
-            Hi, my name is Jonathan!
+            {" "}
```
Comment (minachuong) [inline app/javascript/components/pages/AboutUs.js]: please remove

File: app/javascript/components/pages/AboutUs.js [hunk 2]
```diff
@@ -84,30 +96,30 @@ const AboutUs = (props) => {
...
 //       <>
 //         <h1>About Us</h1>
 //       </>
-    
-         
-//       )    
+
+//       )
 //   }
 // }
```
Comment (minachuong) [inline app/javascript/components/pages/AboutUs.js]: please remove if no longer used

File: app/javascript/components/pages/_tests_/Show.test.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+import Adapter from "enzyme-adapter-react-16";
+import RecipeShow from "../RecipeShow";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe("When Show renders", () => {
+  it("displays recipe", () => {
+    const show = shallow(<RecipeShow />);
+    const showPage = show.find("CardTitle");
+    expect(showPage.length).toEqual(1);
```
Comment (minachuong) [inline app/javascript/components/pages/_tests_/Show.test.js:12]: Expectations on the title text would build stronger tests

**Review by minachuong — APPROVED** [review]
👍

File: app/javascript/components/pages/_tests_/Show.test.js (hunk 3 above)
Comment (csullivan710) [reply app/javascript/components/pages/_tests_/Show.test.js:12]: I think he started this just to see how testing would go

**Pull request merged**